### DISCUSSION TITLE
Redash Initial Setup not shown after deployment to Heroku

### DISCUSSION THREAD

**Discussion by jkwuc89** [getredash/redash#5983] [Q&A] [answered]
According to https://redash.io/help/open-source/setup#setup-redash-instance-setup, Redash will show the Redash Initial Setup page after Redash is up and running. After deploying a new instance of Redash to Heroku using Docker, it is not going to the initial setup page. Instead, it is going to the standard login page. Is there a way to force Redash to go to the initial setup page?

Please let me know if more information is required to answer my question above.

**Comment by justinclift** [comment, marked as answer]
In theory, the initial setup page should only be skipped if there's some form of initial user/organisation in the system.

I'd probably take a look in the PostgreSQL database in your Heroku Redash setup, and see what kind of user/organisation is in there.  Maybe something got pre-populated?

**Comment by justinclift** [comment]
@jkwuc89 It sounds like that turned out to be the problem?
